src/@chakra-ui/gatsby-plugin/foundations/colors.ts

```diff
@@ -1,7 +1,7 @@
 export type Colors = typeof colors
 
 const colors = {
-  grey: {
+  gray: {
```

Comment:
I see in the figma this is spelt `grey`. Should that be updated there to reflect this change. I have no strong preference on the spelling we use, just should be consistent.

Reply:
Good call, yes.

To clarify, Chakra is using `gray`. Calling @nloureiro to update those names in the design system and keep the consistency.